job-service-db/pom.xml

```diff
@@ -34,13 +34,15 @@
         <dependency>
             <groupId>com.github.cafapi.util</groupId>
             <artifactId>util-liquibase-installer</artifactId>
+            <version>1.16.0-SNAPSHOT</version>
```

Comment:
Is there something from this version that was needed? This version must have previously been managed from a parent project using dependency management, does this need changed instead? Same with the plugin version below.

Reply:
I agreed - better to update whatever `caf-common` is referenced.

Reply:
Plugin version was needed to pull the 1.16.0-SNAPSHOP Version. Without plugin change, it was pulling the 1.15.0-1.  Refactored the code in `util-liquibase-installer` to take full connection URL, that's the way we are passing from marathon template. For that to work, I had to make code change.  Otherwise we had to make code other repos to make it work with 1.15.0-1